rust/ql/lib/codeql/rust/elements/internal/CallExprImpl.qll

```diff
@@ -21,5 +21,7 @@ module Impl {
    * foo(1) = 4;
    * ```
    */
-  class CallExpr extends Generated::CallExpr { }
+  class CallExpr extends Generated::CallExpr {
+    override string toString() { result = "... (...)" }
```

Comment:
Here it would be really nice to be able to see the identifier/path when the invoked expression is just an identifier/path expression 🙏 😀

Reply:
Thanks @redsun82 😍